test/helpers/quictestlib.c

```diff
@@ -503,6 +506,23 @@ int qtest_create_quic_connection_ex(QUIC_TSERVER *qtserv, SSL *clientssl,
 
     if (clientssl != NULL)
         abortserverthread = 0;
+
+    client_ready_cond = ossl_crypto_condvar_new();
```

Comment:
shall we do something like
```
if (client_ready_cond == NULL) {
    client_ready_cond = ossl_crypto_condvar_new();
    ....
}
```

the thing is we create the mutex and condvar here in a main thread (a.k.a. client).

later we spawn a server thread at line 539, which executes the same function. The server thread then is going to create the  pair of condvar/mutex. May be I'm missing something fundamental here.